src/proof.rs

```diff
@@ -517,7 +518,6 @@ impl<'a, 'b> ProofTreePrinterImpl<'a, 'b> {
     fn fmt_compressed(&mut self) -> fmt::Result {
         let parents = self.p.arr.count_parents();
         let rpn = self.p.arr.to_rpn(&parents, false);
-        let mut proof_ordered_hyps = vec![];
```

Comment:
`proof_ordered_hyps` seemed to gather hypotheses. However here they were gathered in the order they were needed in the RPN steps. I'm not gathering them here anymore.